auth0/env.bash

```
@@ -0,0 +1,7 @@
+#!/usr/bin/env bash
+
+TENANT=
```

Shellcheck (0.10.0)

[warning] TENANT appears unused. Verify use (or export if used externally).

(SC2034)